src/GafferSceneUIModule/LightEditorBinding.cpp

```diff
@@ -355,6 +366,104 @@ CompoundDataPtr MuteColumn::m_muteUndefinedIconData = new CompoundData(
 
 StringDataPtr MuteColumn::m_muteBlankIconName = new StringData( "muteLightUndefined.png" );
 
+class SetMembershipColumn : public InspectorColumn
+{
+
+	public :
+		IE_CORE_DECLAREMEMBERPTR( SetMembershipColumn )
+
+		SetMembershipColumn( const GafferScene::ScenePlugPtr &scene, const Gaffer::PlugPtr editScope, const IECore::InternedString &setName, const std::string &columnName )
+			: InspectorColumn( new GafferSceneUI::Private::SetMembershipInspector( scene, editScope, setName ), columnName ), m_setName( setName )
+		{
+
+		}
+
+		CellData cellData( const Gaffer::Path &path, const IECore::Canceller *canceller ) const override
+		{
+			CellData result = InspectorColumn::cellData( path, canceller );
+
+			auto scenePath = runTimeCast<const ScenePath>( &path );
+			if( !scenePath )
+			{
+				return result;
+			}
+
+			std::string toolTip;
+			if( auto toolTipData = runTimeCast<const StringData>( result.toolTip ) )
+			{
+				toolTip = toolTipData->readable();
+			}
+
+			if( auto value = runTimeCast<const IntData>( result.value ) )
+			{
+				if( value->readable() & PathMatcher::Result::ExactMatch )
+				{
+					result.icon = m_soloIconData;
+				}
+				else if( value->readable() & PathMatcher::Result::AncestorMatch )
+				{
+					ConstPathMatcherDataPtr soloSetData = scenePath->getScene()->set( m_setName );
+					const PathMatcher &soloSet = soloSetData->readable();
+
+					ScenePlug::ScenePath currentPath( scenePath->names() );
+					while( !currentPath.empty() )
+					{
+						if( soloSet.match( currentPath ) & PathMatcher::Result::ExactMatch )
+						{
+							result.icon = m_soloIconFadedData;
+							toolTip = "Inherited from : " + ScenePlug::pathToString( currentPath );
+							break;
+						}
+						currentPath.pop_back();
+					}
+				}
+			}
+
+			if( !result.icon )
+			{
+				result.icon = m_soloUndefinedIconData;
+			}
+
+			result.value = nullptr;
+			result.toolTip = new StringData( toolTip + ( toolTip.size() ? "\n\n" : "" ) + "Double-click to toggle" );
+
+			return result;
+		}
+
+		CellData headerData( const IECore::Canceller *canceller ) const override
+		{
+			return CellData( new StringData( "Solo" ) );
+		}
```

Comment:
This is hardcoded to `Solo` still. I think we can just remove the `headerData()` override entirely and let the base class do it?

Reply:
Actually I think we still want the `headerData()` override because it let's us add an icon to the header when something is soloed. I had forgotten that part before, added in https://github.com/GafferHQ/gaffer/pull/5108/commits/3896da0c85f7f5d811b0b09bc88c15e22f660a41

I'm using a transparent icon to fill the space and prevent a column resize when the green dot goes on or off. Is there a better solution you know of? I didn't see a way to set the minimum width, but perhaps I'm missing it.

Reply:
And for bookkeeping, the hardcoded "Solo" title was fixed in https://github.com/GafferHQ/gaffer/pull/5108/commits/6daee8713a5a1d1daefc52552264d8b994061b5c